src/artikler/andre-muligheter/JobblystPanel.tsx

```diff
@@ -11,10 +11,17 @@ export const JobblystBokmalPanel = () => {
             border={false}
             className="jobblyst_panel"
         >
-            <Innholdstittel tag="h2">Jobblyst på Facebook</Innholdstittel>
-            <Normaltekst>
-                Råd og veiledning for deg som er ung og har spørsmål
-            </Normaltekst>
+            <div className="jobblyst_panel__content">
```

Comment:
Bare nysgjerrig og vil lære: 
Hvorfor drar vi ikke denne ut i en egen `<Jobblyst>` komponent som kun tar inn en tekst som props?

Reply:
@siljee  Man kunne godt laget en <Jobblyst> komponent. Samtidig når komponenten er så enkel, så sparer man ikke så mange linjer med kode og man må slå opp i en av fil for å se hva slags html som blir generert.

Reply:
Er ikke uenig i at vi kan trekke ut dette til et eget komponent. Samtidig så ønsker vi jo kanskje å trekke ut tekster til Sanity, og da er det mer relevant å se på disse endringene da?

Reply:
Ja, da skjønner jeg at ingenting er feil. Men det er en avveiningssak. 
Good good 👍🏻